tests/common.go

```diff
@@ -5317,6 +5317,157 @@ func GetSubsetOfSlice[T any](items []T, length int) ([]T, error) {
 	return randomItems, nil
 }
 
+// MakeStoragetoStoragelessNode returns true on converting Storage Node to Storageless Node
+func MakeStoragetoStoragelessNode(n node.Node) error {
+	storageLessNodeBeforePoolDelete := node.GetStorageLessNodes()
+	// Get total list of pools present on the node
+	poolList, err := GetPoolsDetailsOnNode(n)
+	if err != nil {
+		return err
+	}
+
+	lenPools := len(poolList)
+	log.InfoD("total number of Pools present on the Node [%v] is [%d]", n.Name, lenPools)
+
+	// Enter pool maintenance mode before deleting the pools from the cluster
+	err = Inst().V.EnterPoolMaintenance(n)
+	if err != nil {
+		return fmt.Errorf("failed to set pool maintenance mode on node %s : Error [%v]", n.Name, err)
+	}
+
+	time.Sleep(1 * time.Minute)
+	expectedStatus := "In Maintenance"
+	err = WaitForPoolStatusToUpdate(n, expectedStatus)
+	if err != nil {
+		return fmt.Errorf("node [%s] pools are not in status [%s] : Error [%v]", n.Name, expectedStatus, err)
+	}
+
+	// Delete all the pools present on the Node
+	for i := 0; i < lenPools; i++ {
+		err := Inst().V.DeletePool(n, strconv.Itoa(i))
+		if err != nil {
+			return err
+		}
+	}
+
+	err = Inst().V.ExitPoolMaintenance(n)
+	if err != nil {
+		return fmt.Errorf("failed to exit pool maintenance mode on node [%s] Error: [%v]", n.Name, err)
+	}
+
+	err = Inst().V.WaitDriverUpOnNode(n, 5*time.Minute)
+	if err != nil {
+		return fmt.Errorf("volume driver down on node %s with Error: [%v]", n.Name, err)
+	}
+	expectedStatus = "Online"
+	err = WaitForPoolStatusToUpdate(n, expectedStatus)
+	if err != nil {
+		return fmt.Errorf("node %s pools are not in status %s with Error: [%v]", n.Name, expectedStatus, err)
+	}
+
+	storageLessNodeAfterPoolDelete := node.GetStorageLessNodes()
+	if len(storageLessNodeBeforePoolDelete) <= len(storageLessNodeAfterPoolDelete) {
+		return fmt.Errorf("making storage node to storagelessnode failed")
+	}
+	return nil
+}
+
+// IsPksCluster returns true if current operator installation is on an EKS cluster
+func IsPksCluster() bool {
+	if stc, err := Inst().V.GetDriver(); err == nil {
+		if oputil.IsPKS(stc) {
+			logrus.Infof("PKS installation with PX operator detected.")
```

Comment:
use log info, logrus will not push to log file or dashboard

Reply:
fixing now